build.cake

```diff
@@ -101,25 +106,31 @@ Task("Externals-Apple")
     .WithCriteria(() => IsRunningOnUnix())
     .Does(() =>
 {
-    CleanDirectory(AppleExternals);
     var zipFile = System.IO.Path.Combine(AppleExternals, "apple.zip");
+    if (FileExists(zipFile))
```

Comment:
This will skip the download part if zip exists. Is there a task which re-downloads the archive?

Reply:
not really. It long task and if you want to really re-download it, you should clean first